go/libkb/secret_store.go

```diff
@@ -64,9 +92,13 @@ func GetConfiguredAccounts(c SecretStoreContext) ([]keybase1.ConfiguredAccount,
 }
 
 func ClearStoredSecret(g *GlobalContext, username NormalizedUsername) error {
-	secretStore := NewSecretStore(g, username)
-	if secretStore == nil {
+	if g.SecretStoreAll == nil {
 		return nil
 	}
-	return secretStore.ClearSecret()
+	return g.SecretStoreAll.ClearSecret(username)
+}
+func NewTestSecretStoreAll(c SecretStoreContext, g *GlobalContext) SecretStoreAll {
```

Comment:
Can this function go in secret_store_test.go so it is only compiled into test builds?